openedx/adg/lms/webinars/models.py

```diff
@@ -120,6 +121,27 @@ class Webinar(TimeStampedModel):
         blank=True,
         related_name='webinar_modified_by',
     )
+    zoom_id = models.CharField(verbose_name=_('Webinar Zoom Id'), max_length=100, blank=True, null=True, )
+    AUDIO_SETTINGS_CHOICES = (
+        ('telephone', _('Telephone')),
+        ('computer_audio', _('Computer Audio')),
+        ('both', _('Both'))
+    )
+    audio_settings = models.CharField(
+        verbose_name=_('Audio Settings Choices'),
+        choices=AUDIO_SETTINGS_CHOICES,
+        max_length=20,
+        default='both',
+    )
+    duration = models.IntegerField(default=0, verbose_name=_('Duration'), help_text=_('Duration in minutes'))
+    host_video = models.BooleanField(default=False, verbose_name=_('Host Video Enabled'), )
+    panelist_video = models.BooleanField(default=False, verbose_name=_('Panelist Video Enabled'), )
```

Comment:
As per the designs, panelists video is on by default. Can we update the default?